testsuite/features/step_definitions/command_steps.rb

```diff
@@ -236,10 +236,14 @@
   $server.run_until_ok("#{cmd} #{system_name} state.highstate")
 end
 
-When(/^I wait until "([^"]*)" service is active on "([^"]*)"$/) do |service, host|
+Then(/^I wait until "([^"]*)" service is (active|inactive) on "([^"]*)"$/) do |service, status, host|
```

Comment:
"I wait" is an action.

If we want "Then" we should rephrase as "XXX service should be active after waiting" (test).